README.md

```diff
@@ -117,3 +117,28 @@ returns
   "error": "<ERROR_MESSAGE>"
 }
 ```
+
+### Set Tasks Complete by ID (Array of IDs) 
+POST /api/tasks/complete_task
+```
+{
+	"ids": [<array_of_task_ids>]
+}
+```
+returns
+```
+{
+    "data": [
+        {
+            "id": <event_id>,
+            "name": "<event_name>",
+            "description": "<event_description>",
+            "duration": <task_duration_in_seconds>,
+            "scheduled": <boolean>,
+            "completed": <boolean>,
+            "timecompleted": "<timestamp>"
+        }
+    ],
+    "error": "<ERROR_MESSAGE>"
+}
```

Comment:
Any chance you could make it clear that it returns an array of task responses? Either with a comment on the returns line or by adding another task in the sample response

Reply:
Yeah I can add another task object to make it clearer.